pkg/minishift/clusterup/clusterup.go

```diff
@@ -41,6 +41,7 @@ import (
 	"regexp"
 
 	"github.com/minishift/minishift/pkg/util"
+	utilStrings "github.com/minishift/minishift/pkg/util/strings"
```

Comment:
imo not a good name. stringUtils might already be better. Or maybe minishiftStrings to make it clear that this is our won string helper